extensions/amp-script/0.1/amp-script.js

```diff
@@ -109,12 +116,16 @@ export class AmpScript extends AMP.BaseElement {
       // `workerUrl` is from CDN, so no need for `ampCors`.
       xhr.fetchText(workerUrl, {ampCors: false}).then(r => r.text()),
       xhr.fetchText(authorUrl).then(r => r.text()),
+      getElementServiceForDoc(this.element, TAG, TAG),
```

Comment:
Shouldn't this be on the `Services` object?

Reply:
Didn't want to expose it since it's basically a "private" service.